programming-core/src/main/java/org/objectweb/proactive/core/httpserver/ClassServerServlet.java

```diff
@@ -132,7 +132,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         if (logger.isDebugEnabled()) {
             final String from = req.getLocalAddr() + ":" + req.getLocalPort();
             final String status = ok ? "OK" : "Class Not Found";
-            logger.debug("Served request from " + from + " for " + className + " status: " + status);
+            if (logger.isDebugEnabled()) {
+                logger.debug("Served request from " + from + " for " + className + " status: " + status +
+                             (bytes != null ? " bytes: " + bytes.length : ""));
```

Comment:
Can you add "bytes array null" instead of empty string "". I think that has valuable information as well. when the bytes array is null..

Reply:
Can you guard the `bytes.length` with `String.valueOf(bytes.length)`

Reply:
bytes array will be null when status is not ok, thus it does not provide more information.